src/main/resources/templates/db_scripts/schema.json

```diff
@@ -177,6 +177,10 @@
           "fieldName" : "personal.lastName",
           "tOps" : "ADD"
         },
+        {
+          "fieldName" : "personal.preferredFirstName",
+          "tOps" : "ADD"
+        },
```

Comment:
Is there a test for searching by preferred first name?

If there isn't, please add one.

Reply:
There is one now.

Reply:
Thank you.